xpallets/system/src/lib.rs

```diff
@@ -62,8 +62,8 @@ decl_storage! {
         /// Paused pallet call.
         pub Paused get(fn paused): map hasher(twox_64_concat) Vec<u8> => BTreeMap<Vec<u8>, ()>;
 
-        /// Blocked account ids.
-        pub BlockedAccounts get(fn blocked_accounts): map hasher(blake2_128_concat) T::AccountId => bool;
+        /// The accounts that are blocked.
+        pub Blacklist get(fn blacklist): map hasher(blake2_128_concat) T::AccountId => bool;
```

Comment:
I prefer to use `Blocklist` instead of `Blacklist`

Reply:
Since it's already used in Substrate and `Blocklist` is not a _valid_ word in the dictionary, I'll keep it for now.